crates/biome_js_analyze/src/lint/nursery/no_useless_catch_binding.rs

```diff
@@ -0,0 +1,97 @@
+use crate::JsRuleAction;
+use crate::lint::correctness::no_unused_variables::is_unused;
+use crate::services::semantic::Semantic;
+use biome_analyze::{FixKind, Rule, RuleDiagnostic, context::RuleContext, declare_lint_rule};
+use biome_console::markup;
+use biome_js_syntax::{JsCatchDeclaration, binding_ext::AnyJsIdentifierBinding};
+use biome_rowan::{AstNode, BatchMutationExt};
+use biome_rule_options::no_useless_catch_binding::NoUselessCatchBindingOptions;
+
+declare_lint_rule! {
+    /// Disallow unused catch bindings.
+    ///
+    /// This rule enforces removing unnecessary catch bindings in accordance with ECMAScript 2019.
```

Comment:
The rule disallows the binding, the removing is just a code action. I would reword this paragraph to put emphasis on the disallow part